src/components/AssetInfo/TokenSideBarHolder.tsx

```diff
@@ -24,11 +24,12 @@ const TokenSideBarHolder = ({
   surrenderDocument
 }: TokenSideBarHolderProps) => {
   const showChangeBeneficiary = !!approvedBeneficiaryAddress || isEqualBeneficiaryAndHolder;
+  const isEndorseChangeOfBeneficiaryAwaiting = !!approvedBeneficiaryAddress && !isEqualBeneficiaryAndHolder;
```

Comment:
can remove the awaiting from varible name. or just name it isApprovedBeneficiaryAddress.